apps/sv/frontend/src/components/votes/actions/CreateUnallocatedUnclaimedActivityRecord.tsx

```diff
@@ -0,0 +1,155 @@
+// Copyright (c) 2024 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import { DateWithDurationDisplay } from '@lfdecentralizedtrust/splice-common-frontend';
+import { Decimal } from 'decimal.js-light';
+import React, { useCallback, useEffect, useState } from 'react';
+import { FormControl, Stack, TextField, Typography } from '@mui/material';
+import { DesktopDateTimePicker } from '@mui/x-date-pickers/DesktopDateTimePicker';
+import { LocalizationProvider } from '@mui/x-date-pickers';
+import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
+import dayjs, { Dayjs } from 'dayjs';
+import utc from 'dayjs/plugin/utc';
+
+import { getUTCWithOffset } from '@lfdecentralizedtrust/splice-common-frontend-utils';
+import { ActionRequiringConfirmation } from '@daml.js/splice-dso-governance/lib/Splice/DsoRules/module';
+import { ActionFromForm, actionFromFormIsError } from '../VoteRequest';
+
+dayjs.extend(utc);
+
+function asCreateUnallocatedUnclaimedActivityRecord(action?: ActionFromForm):
+  | {
+      beneficiary: string;
+      amount: string;
+      reason: string;
+      expiresAt: string;
+    }
+  | undefined {
+  return action &&
+    !actionFromFormIsError(action) &&
+    action.tag === 'ARC_DsoRules' &&
+    action.value.dsoAction.tag === 'SRARC_CreateUnallocatedUnclaimedActivityRecord'
+    ? action.value.dsoAction.value
+    : undefined;
+}
+
+const CreateUnallocatedUnclaimedActivityRecord: React.FC<{
```

Comment:
It seems like we are currently missing a check that the amount is <= the amount of unclaimed rewards available. We should add that. In the end the Daml code ofc enforces this but we want to guard against obvious operator errors.

I'm ok with doing that in a separate PR. If so, let's create an issue and add a TODO here.